OSVR-Unity/Assets/OSVRUnity/src/ClientKit.cs

```diff
@@ -78,7 +85,20 @@ private void EnsureStarted()
             void Awake()
             {
                 DLLSearchPathFixer.fix();
-                DontDestroyOnLoad(gameObject);
+                //if an instance of this singleton does not exist, set the instance to this object and make it persist
+                if(_instance == null)
+                {
+                    _instance = this;
+                    DontDestroyOnLoad(this);
```

Comment:
Here you have `DontDestroyOnLoad(this);` while in the property getter you have effectively `DontDestroyOnLoad(this.gameObject)` - is this intentional, harmless, or a bug?